src/main/java/Duke.java

```diff
@@ -1,10 +1,117 @@
+import DukePackage.ChatUI;
+import DukePackage.Parser;
+import DukePackage.Storage;
+import DukePackage.Task;
+import DukePackage.TaskType;
+import java.util.Objects;
+
+
+/**
+ * The Duke class is the main class that runs the Duke chatbot application.
+ */
 public class Duke {
+
+    /**
+     * The main method that starts the Duke application.
+     * @param args The command-line arguments passed to the application.
+     */
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
```

Comment:
Do you want to print the logo using ChatUI class instead?

Reply:
oops Thank you. I missed this out and forgot to remove logo. Current UI doesn't include logo